### PR TITLE
Silent below steer speed alert

### PR DESCRIPTION
**Pull request by sshane**

### CODE REVIEW

File: selfdrive/car/chrysler/carcontroller.py [hunk 1]
```diff
@@ -57,6 +57,8 @@ def update(self, CC, CS):
       elif self.CP.carFingerprint in RAM_CARS:
         if CS.out.vEgo < (self.CP.minSteerSpeed - 0.5):
           lkas_control_bit = False
+      if CS.out.steerFaultTemporary or CS.out.steerFaultPermanent:
+        lkas_control_bit = False
```
Comment (sshane) [inline selfdrive/car/chrysler/carcontroller.py:61]:
```suggestion
```

File: selfdrive/controls/lib/events.py [hunk 2]
```diff
@@ -238,7 +238,7 @@ def below_steer_speed_alert(CP: car.CarParams, CS: car.CarState, sm: messaging.S
     f"Steer Unavailable Below {get_display_speed(CP.minSteerSpeed, metric)}",
     "",
     AlertStatus.userPrompt, AlertSize.small,
-    Priority.MID, VisualAlert.steerRequired, AudibleAlert.prompt, 0.4)
+    Priority.MID, VisualAlert.steerRequired, AudibleAlert.none, 0.4)
```
Comment (sshane) [inline selfdrive/controls/lib/events.py:241]: Might want to only mute sub-ten as discussed in Discord

**Comment by adeebshihadeh** [comment]
I agree this needs to be fixed, but I'm not really sure this is better. One of the biggest complaints with the EV6 stock system is that the steering cuts out without warning. Let's keep this open for now, and I'll think about another solution while working on some of the planned UI updates.

**Comment by sshane** [comment]
Think an issue is more appropriate for this

**Pull request closed without merging**